group_vars/repec/repec_staging.yml

```diff
@@ -3,7 +3,10 @@ passenger_server_name: "repec-staging1.princeton.edu"
 passenger_app_env: "staging"
 passenger_ruby: "/usr/bin/ruby2.7"
 ruby_version_override: "ruby2.7"
-db_host: "maria-staging2.princeton.edu"
+
+postgres_host: "lib-postgres3.princeton.edu"
+
```

Comment:
@christinach yes.  Since we are not installing postgress, we are just utilizing the postgres version install on the server. I do not think setting it changes the results any.